bench/00_transform/cub.cu

```diff
@@ -0,0 +1,80 @@
+#include "matx.h"
+#include <nvbench/nvbench.cuh>
+
+
+using namespace matx;
+
+//
+// types of data to run tests on
+//
+// using sort_types = nvbench::type_list<matxFp16Complex, cuda::std::complex<float>, cuda::std::complex<double>, float, double>;
+using sort_types = nvbench::type_list<double>;
+
+////////////////////////////////////////////////////////////////////////////////
+///
+///\brief benchmark for 1D Sorts
+///
+///\param param in
+///
+////////////////////////////////////////////////////////////////////////////////
+template <typename ValueType>
+void sort1d_range(
```

Comment:
You can probably remove "range" from the name since it's implied in the range that runs